docs/contributing/reporting-bugs.md

```diff
@@ -118,9 +118,9 @@ reproducible bug, a GitHub issue will be filed to keep track of it (see below).
 
 ## Managing bug reports
 
-This section describes our bug management process. All community members are
-encouraged to help make sure things run smoothly, whether or not they originally
-reported the bug.
+This section describes our bug-management process. All community members are
```

Comment:
Hmm, I'm not sure bug-management is a word? How about just "This describes our process for managing bugs."?

Reply:
Before the noun, "bug management" functions as a compound adjective, aka a phrasal adjective :) But if you prefer the later phasing, I can fix it up.

Reply:
Yeah I think I prefer the latter phrasing anyway.